mopidy_local/commands.py

```diff
@@ -143,19 +147,54 @@ def _check_tracks_in_library(
         return files_to_update, files_in_library
 
     def _find_files_to_scan(
-        self, *, media_dir, file_mtimes, files_in_library, excluded_file_exts
+        self,
+        *,
+        media_dir,
+        file_mtimes,
+        files_in_library,
+        included_file_exts,
+        excluded_file_exts,
     ):
         files_to_update = set()
 
+        def _is_hidden_file(relative_path, file_uri):
```

Comment:
*When* we come to finally add tests here, we'll want these functions defined outside of `_find_files_to_scan`. But we can fix that when we get there.